inc/fun/shortcode.php

```diff
@@ -0,0 +1,704 @@
+<?php
```

Comment:
`shortcode.php` -> `shortcodes.php`

Reply:
resolve in https://github.com/solstice23/argon-theme/pull/478/commits/84f3020c014f9232e2dfa5da926a4fc16904d15c, https://github.com/solstice23/argon-theme/pull/478/commits/319b53d04d1d3ced302e6840d563c4220741d748 .